src/api/config/security.py

```
@@ -0,0 +1,85 @@
+        },
+        'loggers': {
+            'security': {
+                'handlers': ['security_file'],
+                'level': 'INFO',
+            },
+            'access': {
+                'handlers': ['access_file'],
+                'level': 'INFO',
+            }
+        }
+    } 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)